CITATION.cff

```diff
@@ -0,0 +1,34 @@
+cff-version: 1.2.0
+message: "If Python-Stratify played an important part in your research then please add us to your reference list by using the references below."
+authors:
+- family-names: "Elson"
+  given-names: "Phil"
+  orcid: "https://orcid.org/0000-0001-6360-749X"
+- family-names: "Little"
+  given-names: "Bill"
+  orcid: "https://orcid.org/0000-0002-1345-9465"
+- family-names: "Andela"
+  given-names: "Bouwe"
+  orcid: "https://orcid.org/0000-0001-9005-8940"
+- family-names: "Dreyer"
+  given-names: "Laura"
+  orcid: "https://orcid.org/0009-0006-6310-3658"
+- family-names: "Fernandes"
+  given-names: "Filipe"
+  orcid: "https://orcid.org/0000-0003-4165-2913"
+- family-names: "Rol"
+  given-names: "Evert"
+  orcid: "https://orcid.org/0000-0001-8357-4453"
+- family-names: "Wright"
+  given-names: "Henry"
+  orcid: "https://orcid.org/0009-0005-1489-760X"
+- family-names: "Yeo"
+  given-names: "Martin"
+  orcid: "https://orcid.org/0009-0000-6857-560X"
+title: "python-stratify"
+abstract: "Interpolation for restratification, particularly useful for Nd vertical interpolation of atmospheric and oceanographic datasets"
+license: "BSD-3-Clause"
```

Comment:
```suggestion
license: "BSD-3-Clause"
license-url: "https://spdx.org/licenses/BSD-3-Clause.html"
```

Reply:
Thanks Bill for the review and suggestion. see 307195f